cardano-db-sync/src/Cardano/DbSync/Cache/Types.hs

```diff
@@ -45,6 +47,11 @@ import Ouroboros.Consensus.Cardano.Block (StandardCrypto)
 
 type StakePoolCache = Map PoolKeyHash DB.PoolHashId
 
+data StakeCache = StakeCache
```

Comment:
Is it possible to have a little code comment about the reasoning for having 2 caches?